### ISSUE TITLE
AudioAddict API

### ISSUE THREAD

**Issue by GertBurger** [Bitcrusher/Digitally-Imported-XBMC-plugin#14]
Hi

I see AudioAddict actually has an unpublished API which someone documented at http://difm.eu/dox/

Has use of this been considered before?

I wouldn't mind writing the code but was just wondering there was maybe another reason its not used.

**Comment by Bitcrusher** [comment]
Hi Gert
Yes, I was made aware of their API before i started making v2 of the plugin and I was actually talking to the DI staff, about getting "legit" access and documentation for it, but it took longer and longer for them to respond and the communications just died out.

As I back then couldn't find any documentation for the API, back then, I decided not to wait anymore and rewrote the script and haven't looked back since.

I did get a patch at one point, a guy that added authentication through their API which it/they (DI/Radiotunes/Jazzradio) now use.

When I get around to another rewrite/v4, I'll definitely look into using the whole API now that someone else has made the documentation available for it :)
But if I remember correctly, the API requires (at least back then) the user to register, also if your using free streams, which I think would be a hassle/annoying for most people - if that's still the case, I don't know.

I've just released v3.0.6 that fixes the plugin again, after DI.fms recent redesign.
